syntax = "proto2";

message UploadResp {
	required int64 state = 1;
}
